tests/testsuite.src/used_binaries.at

```diff
@@ -918,3 +912,62 @@ AT_CHECK([cat prog.cob | $COMPILE_MODULE -j -], [0], [job], [],
 )
 
 AT_CLEANUP
+
+
+AT_SETUP([cobc diagnostics show caret])
+# promoted on 2023-06-01T09:57
+AT_KEYWORDS([cobc diagnostics])
+AT_DATA([prog.cob],[
+       IDENTIFICATION   DIVISION.
+       PROGRAM-ID.      prog.
+       DATA             DIVISION.
+       WORKING-STORAGE  SECTION.
+       01 TEST-VAR PIC 9(2) VALUE 'A'.
+       COPY 'CRUD.CPY'.
+       PROCEDURE        DIVISION.
+           DISPLAY TEST-VAR NO ADVANCING
+           END-DISPLAY
+           MOVE 12 TO TEST-VAR
+           DISPLAY TEST-VAR NO ADVANCING
+           END-DISPLAY
+           STOP RUN.
+])
+AT_CHECK([$COBC -Wall -fsyntax-only prog.cob], [1], [], [prog.cob:7: error: CRUD.CPY: No such file or directory
```

Comment:
this line should be:

```
AT_CHECK([$COMPILE_ONLY prog.cob], [1], [],
[prog.cob:7: error: CRUD.CPY: No such file or directory
```

the following test should be

```
AT_CHECK([$COMPILE_ONLY -fdiagnostics-show-caret -fdiagnostics-show-line-numbers prog.cob], [1], [],
[prog.cob:7: error: CRUD.CPY: No such file or directory
```

and the last (which may be switched with the second)

```
AT_CHECK([$COMPILE_ONLY -fdiagnostics-show-caret -prog.cob], [1], [],
[prog.cob:7: error: CRUD.CPY: No such file or directory
```